File: 4281-score-validator/score-validator.cpp
class Solution {
public:
    vector<int> scoreValidator(vector<string>& events) {
        int score=0,count=0;

        for (auto ch: events){
            if (ch=="W")
                count+=1;

            else if (ch=="WD" or ch=="NB")
                score+=1;

            else
                score+=stoi(ch);

            if (count==10) break;
        }

        return {score,count};
    }
};

File: 4281-score-validator/score-validator.py
class Solution:
    def scoreValidator(self, events: list[str]) -> list[int]:
        score=count=0

        for ch in events:
            if ch=='W':
                count+=1

            elif ch=='WD' or ch=='NB':
                score+=1

            else:
                score+=int(ch)

            if count==10: break

        return [score,count]